src/style.css

```diff
@@ -1,3 +1,73 @@
-body {
+* {
   margin: 0;
+  font-family: "Roboto", sans-serif;
+}
+
+html {
+  --active-link: #00acdc;
+}
+
+.header {
+  background-color: white;
+  display: flex;
+  justify-content: space-between;
+  min-width: 1024px;
+  max-width: 1200px;
+}
+
+.header img {
+  display: block;
+  margin-left: 50px;
+  padding: 10px 0;
+}
+
+.nav__item {
+  display: flex;
+  margin-right: 50px;
+}
+
+.nav__list {
+  display: flex;
+  list-style: none;
+}
+
+.nav__item li {
```

Comment:
the 'li' element should have the nav__item class
the 'ul' element should have the nav__list class
the 'a' element should have the nav__link class

Consider avoiding tag usage, especially if it's not a general (page level) style or reset styling